apollo-ios-codegen/Sources/GraphQLCompiler/JavaScript/package.json

```diff
@@ -6,27 +6,27 @@
   "license": "MIT",
   "author": "Apollo GraphQL <opensource@apollographql.com>",
   "scripts": {
-    "build": "rollup --config",
+    "build": "rollup --config --bundleConfigAsCjs",
     "test": "jest"
   },
   "engines": {
     "npm": ">=7"
   },
   "dependencies": {
-    "graphql": "17.0.0-alpha.3",
+    "graphql": "17.0.0-alpha.2",
```

Comment:
This is the solution to fixing the Node error in the description. I do not believe we need anything in `alpha.3` - all our codegen tests still pass.